### PR TITLE
WIP: add support for cloudwatch log events + api gateway logs

### PR DESCRIPTION
**Pull request by kkleemola**

### CODE REVIEW

File: src/main/java/com/mozilla/secops/parser/Event.java [hunk 1]
```diff
@@ -229,6 +230,25 @@ private static ObjectMapper getObjectMapper() {
...
+   * @param project log group string
+   */
+  public void setCloudWatchLogGroup(String logGroup) {
+    cloudwatchLogGroup = logGroup;
+  }
+
+  /**
+   * Get Cloudwatch log group
+   *
+   * @return Stackdriver project name, or null if was not present
```
Comment (ameihm0912) [inline src/main/java/com/mozilla/secops/parser/Event.java]: `Stackdriver project name` -> `Log group`?

File: src/main/java/com/mozilla/secops/parser/Event.java [hunk 2]
```diff
@@ -229,6 +230,25 @@ private static ObjectMapper getObjectMapper() {
...
+   */
+  @JsonProperty("cloudwatchLogGroup")
```
Comment (ameihm0912) [inline src/main/java/com/mozilla/secops/parser/Event.java]: For consistency likely want `cloudwatch_log_group` here

File: src/main/java/com/mozilla/secops/parser/models/apigateway/ApiGatewayAccessLog.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  public String getUser() {
+    return user;
+  }
+
+  @JsonProperty("requestTime")
+  public String getRequestTime() {
+    return requestTime;
+  }
+
+  @JsonProperty("httpMethod")
```
Comment (ameihm0912) [inline src/main/java/com/mozilla/secops/parser/models/apigateway/ApiGatewayAccessLog.java]: Similar to previous comment about case consistency here

**Comment by ameihm0912** [comment]
@kkleemola just wondering about this PR, is this one we want to fix up and merge?

**Comment by kkleemola** [comment]
> @kkleemola just wondering about this PR, is this one we want to fix up and merge?

I don't think it's worth it given the new architecture. Some of the parser payloads may be useful at some point but they'd be dead code now. I'll close but leave the branch for now.

**Pull request closed without merging**